files/default/lib/bumpzone.rb

```diff
@@ -0,0 +1,112 @@
+#!/opt/chef/embedded/bin/ruby
+require 'git'
+require 'json'
+require 'octokit'
+
+# Library to bump DNS zones
+class BumpZone
+  @commit_msg = "\n\n"
+  @warning_msg = "Warning:\n\n"
+  @warning = false
+  ONELINE_REGEX = /(^[\w\d\.\-]+\s+[\d]+\s+IN\s+SOA.*[\w\d\.\-]+\s+)(\d{10})(\s+[\d]+\s+[\d]+\s+[\d]+\s+[\d]+)/
+  MULTILINE_REGEX =
+    /(^[\w\d\.\-]+\s+IN\s+SOA\s+[\w\d\.\-]+\s+[\w\d\.\-]+\s+\(\s?\n\s+)(\d{10})(.*\n.*\n.*\n.*\n.*\n\s+\))/
+
+  class << self
+    attr_reader :commit_msg
+  end
+
+  def self.add_commit_msg(msg)
+    @commit_msg << msg
+  end
+
+  def self.add_warning_msg(msg)
+    @warning = true
+    @warning_msg << msg
+  end
+
+  def self.regex(file, regex)
+    !(file !~ regex)
+  end
+
+  def self.soa_oneline?(zone)
+    if regex(zone, ONELINE_REGEX)
+      true
+    elsif regex(zone, MULTILINE_REGEX)
+      false
+    end
+  end
+
+  def self.pr_merged(json)
+    if json['action'] == 'closed' && json['pull_request']['merged'] == true
+      nil
+    else
+      puts 'Not a merged PR, skipping...'
+      exit 0
+    end
+  end
+
+  def self.changed_files(json)
+    github = Octokit::Client.new(access_token: ENV['GITHUB_TOKEN'])
+    repo_path = json['repository']['full_name']
+    issue_number = json['number']
+    github.pull_request_files(repo_path, issue_number)
+  end
+
+  def self.git_push
+    # Set up the git gem
+    git = Git.open('.')
+    # Commit changes
+    git.add(all: true)
+    git.commit("Automatically bumping zone serials#{BumpZone.commit_msg}")
```

Comment:
Do you want a space in here maybe? 
'"Automatically bumping zone serials #{BumpZone.commit_msg}")'